clients/fossid-webapp/build.gradle.kts

```diff
@@ -27,7 +29,10 @@ plugins {
 dependencies {
     api(project(":model"))
 
-    api("com.squareup.retrofit2:retrofit:$retrofitVersion")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
```

Comment:
This block needs to go below line 37, to keep `implementation` dependencies together, but still separate `project` and non-`project` dependencies by a blank line.

Reply:
And I believe `retrofit` needs to stay an `api` dependency as the `Call` return type used in `FossIdRestService` comes from `retrofit`.